pkg/frontend/mysql_cmd_executor.go

```diff
@@ -68,6 +68,10 @@ func parameterModificationInTxnErrorInfo() string {
 	return "Uncommitted transaction exists. Please commit or rollback first."
 }
 
+func abortTransactionErrorInfo() string {
```

Comment:
can this be a const?

Reply:
The error info is defined by @domingozhang . The initial character in the error info is capital. The static code checker will pitch an error on the situatin. So defining a function can suppress the error